llvm/include/llvm/Transforms/Instrumentation/PGOInstrumentation.h

```diff
@@ -56,12 +56,14 @@ class PGOInstrumentationGenCreateVar
 /// The instrumentation (profile-instr-gen) pass for IR based PGO.
 class PGOInstrumentationGen : public PassInfoMixin<PGOInstrumentationGen> {
 public:
-  PGOInstrumentationGen(bool IsCS = false) : IsCS(IsCS) {}
+  PGOInstrumentationGen(bool IsCS = false, bool IsCtxProf = false)
```

Comment:
Can you convert this to an enum to explicitly callout the type of instrumentation instead of using booleans? Something like --
```
enum InstrumentationType {
  FDO =  0,
  CSFDO,
  CTXPROF
};
```

Reply:
good point. done.